examples/trials/mnist-sharedstorage/config_azureblob.yml

```diff
@@ -24,12 +24,14 @@ trainingService:
                             #   C:/Users/USERNAME/.conda/envs/ENVNAME;C:/Users/USERNAME/.conda/envs/ENVNAME/Scripts;C:/Users/USERNAME/.conda/envs/ENVNAME/Library/bin
 sharedStorage:
   storageType: AzureBlob
+  # please set localMountPoint as absolute path and localMountPoint should outside the code directory
+  # because nni will copy user code to localMountPoint
```

Comment:
suggest to validate the provided path in code, not only put it in comment/doc. we can leave it in next release.

Reply:
Relative path validation is in experiment/config/common.py.